README.md

```diff
@@ -88,7 +88,7 @@ HTML, CSS, NodeJS v18, ReactJS
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- Comming soon...
+- [Link](math-magician-codedit334.netlify.app)
```

Comment:
The link to the live demo is broken hence leads to a broken page as shown below. Kindly check and fix it to make your project more professional. :+1: 

![Screenshot from 2023-03-29 10-08-19](https://user-images.githubusercontent.com/65233860/228501660-e70d8040-a064-43eb-897a-3445b9025a58.png)